portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,58 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html, body {
+  height: 100%; 
 }
 
-#greeting-container {
-  margin-top: 20px;
+.wrapper {  
+  display: flex;
```

Comment:
Woot, flex! This is my favorite reference for all the properties: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
Ha I also use that same link as my go to.